packages/Filter/src/helpers/ruleIsBoolean.js

```diff
@@ -0,0 +1,14 @@
+import rules from "../rules";
+
+export default function ruleIsBoolean(comparator) {
+  return [
+    rules.IS,
+    rules.IS_NOT,
+    rules.IS_BLANK,
+    rules.IS_NOT_BLANK,
+    rules.EQUALS,
+    rules.NOT_EQUAL_TO,
+    rules.IS_EMPTY,
+    rules.IS_NOT_EMPTY,
+  ].includes(comparator);
```

Comment:
```js
const booleanRules = [
    rules.IS,
    rules.IS_NOT,
    rules.IS_BLANK,
    rules.IS_NOT_BLANK,
    rules.EQUALS,
    rules.NOT_EQUAL_TO,
    rules.IS_EMPTY,
    rules.IS_NOT_EMPTY,
  ];

export default function ruleIsBoolean(comparator) {
  return booleanRules.includes(comparator);
```

So we don't have to re-instantiate the array on every check. It is not a huge deal, but it is a low hanging fruit nevertheless